modules/custom/az_news/az_news.info.yml

```
@@ -19,11 +19,11 @@ dependencies:
   - text
   - user
   - az_person
-  - az_flexible_page
+  - az_media
   - az_paragraphs
```

Since we moved field storage for

field_az_published.yml
field_az_expiration_date.yml
field_az_attachments.yml
field_az_media_image.yml
field_az_caption.yml
field_az_summary.yml
field_az_short_title.yml
field_az_subheading.yml
into az_core

```suggestion
  - az_core
  - az_media
  - az_paragraphs
```